packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -186,15 +276,75 @@ const isProofsOfEvidenceDue = (appealCaseData) => {
 
 /**
  * @param {AppealCaseWithAppellant} appealCaseData return object from database call
+ * @returns {boolean}
+ */
+const isAppellantFinalCommentDue = (appealCaseData) => {
+	return appealCaseData.finalCommentsDueDate && !appealCaseData.appellantCommentsSubmitted;
+};
+
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData return object from database call
+ * @returns {boolean}
+ */
+const isAppellantProofsOfEvidenceDue = (appealCaseData) => {
+	return appealCaseData.proofsOfEvidenceDueDate && !appealCaseData.appellantsProofsSubmitted;
+};
+
+/**
+ * @param {AppealCaseWithAppellant | AppealSubmission} appealCaseData return object from database call
  * @returns {string}
  */
 const getAppealType = (appealCaseData) => {
-	if (appealCaseData?.appeal?.state === APPEAL_STATE.DRAFT) {
-		return getAppealTypeName(appealCaseData.appeal.appealType);
+	if (isAppealSubmission(appealCaseData)) {
+		return getAppealTypeName(appealCaseData.appeal?.appealType);
 	}
 	return `${appealCaseData.appealTypeName} appeal`;
 };
 
+/**
+ * @param {AppealCaseWithAppellant} appealCaseData
+ * @returns {boolean}
+ */
+const isEligibilityCompleted = (appealCaseData) => {
+	if (appealCaseData?.appeal?.state === APPEAL_STATE.DRAFT) {
+		const eligibility = appealCaseData.appeal.eligibility;
+		return (
+			eligibility !== undefined &&
+			eligibility.enforcementNotice !== null &&
+			eligibility.applicationDecision !== null &&
+			eligibility.applicationCategories !== null
+		);
+	}
+	return true;
+};
+
+// /**
+//  * @param {AppealCaseWithAppellant} appealCaseData
+//  * @returns {boolean}
+//  */
+// const hasFutureDueDate = (appealCaseData) => {
+// 	const currentDate = new Date();
+// 	return (
+// 		[
+// 			calculateAppealDueDeadline(
+// 				appealCaseData.appealTypeCode,
+// 				appealCaseData.originalCaseDecisionDate
+// 			),
+// 			appealCaseData.statementDueDate,
+// 			appealCaseData.finalCommentsDueDate,
+// 			appealCaseData.proofsOfEvidenceDueDate
+// 		].find((date) => {
+// 			if (!date) return false;
+// 			return new Date(date) > currentDate;
+// 		}) !== undefined
+// 	);
+// };
```

Comment:
🗑️ ?

Reply:
Yep, binning is winning*

*disclaimer - does not always apply, please see T&Cs